python-web-scraping-practical-introduction/mechanicalsoup.py

```diff
@@ -0,0 +1,14 @@
+import time
+import mechanicalsoup
+
+browser = mechanicalsoup.Browser()
```

Comment:
You'll need to rename this file to avoid a circular import that comes from naming the local Python script the same as the Python module you're attempting to import:

```
AttributeError: partially initialized module 'mechanicalsoup' has no attribute 'Browser' (most likely due to a circular import)
```

My suggestion is to name it `mech_soup.py` and for consistency you could also rename the other files, e.g. `beauty_soup.py` and (optional) `regex_soup.py`. 🍜